protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -113,7 +113,7 @@ service TradingDataService {
 
   // List ledger entries
   //
-  // Get a list of ledger entries by asset, market, party, account type and transfer type within the given date range.
+  // Get a list of ledger entries within the given date range.
   // This query requests and sums the number of ledger entries from a given subset of accounts, specified via the 'filter' argument.
   // It returns a time series - implemented as a list of AggregateLedgerEntry structs - with a row for every time
   // the summed ledger entries of the set of specified accounts changes.
```

Comment:
Can we shorten this:
"
Listed entries should be limited to a single party from each side only. If zero or more than one party is provided for each of the sides - sending and receiving accounts, the query returns an error.
"
to just:
"
Each account filter must contain no more than one party ID.